code/obj/item/storage/small_storage_parent.dm

```diff
@@ -299,8 +299,10 @@
 	spawn_contents = list(/obj/item/clothing/mask/breath)
 	make_my_stuff()
 		..()
-		if (prob(15))
+		if (prob(15) || ticker.round_elapsed_ticks > 20 MINUTES)
 			new /obj/item/tank/emergency_oxygen(src)
+		if (ticker.round_elapsed_ticks > 20 MINUTES)
+			new /obj/item/crowbar(src)
```

Comment:
```suggestion
			new /obj/item/crowbar/red(src)
```
This was a suggestion from the forum thread.

Reply:
Sensible.